openai-core/src/commonMain/kotlin/com.aallam.openai.api/message/MessageContent.kt

```diff
@@ -107,7 +107,7 @@ public data class FileCitation(
     /**
      * The specific quote in the file
      */
-    @SerialName("quote") val quote: String,
+    @SerialName("quote") val quote: String? = "",
```

Comment:
```suggestion
    @SerialName("quote") val quote: String? = null,
```
I believe it's better to set `null`as the default :)

Reply:
@AntoD3v could you please apply the request change above? thanks!

Reply:
@aallam Sorry, has the change request been applied now?

Reply:
Yes it is, thank you again! 🙌